code/index.html

```diff
@@ -1,22 +1,137 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css"/>
+    <script src="https://kit.fontawesome.com/4bfe1de3df.js" crossorigin="anonymous"></script>
+    <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
+    <link rel="icon" href="/favicon.ico" type="image/x-icon"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Serif:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&family=Indie+Flower&display=swap" rel="stylesheet">    
+    <title>XOXO Magazine</title>
   </head>
-  <body>
+  <body> 
+     
     <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+      <div class="banner">
+        <marquee behavior="alternate" direction="left"><a href="#">Sign up for more beaty & fashion news</a></marquee>
+      </div> 
```

Comment:
Looks nice! Just be aware that this feature seem to be deprecated. Resource: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/marquee